backend/internal/compile/toolkit/toolkit_prod.go

```diff
@@ -18,7 +18,7 @@ func (e toolkitFsWrapper) Open(name string) (fs.File, error) {
 }
 
 //go:generate rm -rf toolkit
-//go:generate rsync -rv --exclude=node_modules --exclude=robin.servers.json ../../../../toolkit toolkit
+//go:generate rsync -rv --exclude=node_modules --exclude=robin.servers.json ../../../../toolkit/ toolkit
```

Comment:
I found this in the rsync docs: https://linux.die.net/man/1/rsync

```
A trailing slash on the source changes this behavior to avoid
creating an additional directory level at the destination. You can
think of a trailing / on a source as meaning lqcopy the contents of
this directoryrq as opposed to lqcopy the directory by namerq, but
in both cases the attributes of the containing directory are
transferred to the containing directory on the destination. In other
words, each of the following commands copies the files in the same
way, including their setting of the attributes of /dest/foo: 
```